Mathlib/Algebra/Homology/HomotopyCategory/DegreewiseSplit.lean

```diff
@@ -112,9 +111,15 @@ noncomputable def mappingConeHomOfDegreewiseSplitIso :
     have r_f := (σ (p + 1 + 1)).r_f
     have s_g := (σ (p + 1)).s_g
     dsimp at r_f s_g
-    simp [mappingConeHomOfDegreewiseSplitXIso, mappingCone.ext_from_iff _ _ _ rfl,
+    simp only [shiftFunctor_obj_X', mappingConeHomOfDegreewiseSplitXIso, ShortComplex.map_X₂,
```

Comment:
I had left this nonterminal simp. It would be better to replace it with the following code and its output:
```lean
    simp [mappingConeHomOfDegreewiseSplitXIso, mappingCone.ext_from_iff _ _ _ rfl,
      mappingCone.inl_v_d_assoc _ (p + 1) _ (p + 1 + 1) (by linarith) (by linarith),
      cocycleOfDegreewiseSplit, r_f] says

Reply:
I changed it like you suggested. I had to change it in the first place because the nonterminal simp was newly exceeding 200000 heartbeats